api/app/Models/WorkExperience.php

```diff
@@ -230,7 +230,7 @@ public function classification()
     /**
      * Return the department model related to the experience
      */
-    public function department()
+    public function department(): BelongsTo
```

Comment:
Could you please also add the type hint comment like `user()` has to help out Larastan?